### PR TITLE
Breadcrumbs component

### PR DESCRIPTION
**Pull request by peterszerzo**
This PR formalizes the breadcrumbs already live on the showcase page:

screen shot 2018-01-08 at 11 20 40 am

The goal is to support any links (same way Sidenav does it), dropdowns using `<ContextMenu/>`, and whatever else comes up. Would be nice to encourage users to only use this component on the left-hand side of the header, which would give UI's a killer amount of consistency. Thoughts?

### CODE REVIEW

File: packages/components/src/Breadcrumbs.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+const Divider = glamorous.span(({ theme }: { theme: Theme }): {} => ({
+  display: "inline-block",
+  margin: "0 6px",
+  color: theme.colors.black
+}))
+
+export default (props: IProps) => (
+  <Container className={props.className} css={props.css}>
+    {(() => {
```
Comment (TejasQ) [inline packages/components/src/Breadcrumbs.tsx]: Please add some comments here documenting this a bit for newer devs who join us.

**Comment by peterszerzo** [comment]
@TejasQ @micha-f is this a good direction for our breadcrumbs needs in client projects? (in case the context isn't clear, this element appears on the left-hand side of the header, right where the side navigation ends, and replaces the side navigation options we had before)

screen shot 2018-01-08 at 12 24 58 pm

**Comment by TejasQ** [comment]
@peterszerzo, I think it works quite nicely.

**Comment by micha-f** [comment]
@peterszerzo I think it's a good start. We might be able to tweak the styling. The dropdown positioning seems a bit off.

**Comment by peterszerzo** [comment]
@micha-f you're absolutely right - I'll work out the positioning in a later commit.

File: package.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
     "test": "lerna run test",
     "lint": "lerna run lint",
     "prettier": "lerna run prettier",
-    "precommit": "lerna run precommit",
+    "precommit": "lerna run precommit && git add .",
```
Comment (TejasQ) [inline package.json]: I was THIS CLOSE to opening an issue about this 😂

**Comment by peterszerzo** [comment]
The travis machines are systematically giving an `eval yarn` error. Ran identical `ci` checks locally, so it seems like a false alarm. Going ahead with the merge.

**Pull request merged**